Library/Homebrew/download_strategy.rb

```diff
@@ -617,7 +617,7 @@ def fetch
 
   def stage
     super
-    quiet_safe_system "svn", "export", "--force", cached_location, Dir.pwd
```

Comment:
Why are these getting unquieted? In the previous PR you only unquieted those that had `-q` flags.

Reply:
> Why are these getting unquieted?

These are not quiet currently. `quiet_safe_system` replaces i.e. `{ quiet_flag: '--quiet' }`, which only `cvs` is using.